src/index.ts

```diff
@@ -1,5 +1,4 @@
-import { makeDecorator, addons } from '@storybook/addons';
-
+import { makeDecorator, addons } from '@storybook/preview-api';
```

Comment:
I don't see `@storybook/preview-api` in the package.json.  I guess it needs to be added as a peer dependency?

Reply:
Other storybook add-on packages have this as a non-peer dependency, so I'll do that.